src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php

```diff
@@ -0,0 +1,257 @@
+        );
+    }
+
+    /**
+     * @Then I should see :customerName, :street, :postcode, :city, :countryName billing address
+     */
+    public function iShouldSeeBillingAddress($customerName, $street, $postcode, $city, $countryName)
+    {
+        Assert::true(
+            $this->showPage->hasBillingAddress($customerName, $street, $postcode, $city, $countryName),
+            sprintf('Cannot find shipping address "%s, %s %s, %s".', $street, $postcode, $city, $countryName)
+        );
+    }
+
+    /**
+     * @Then I should see :shippingMethodName shipment
+     */
+    public function iShouldSeeShipment($shippingMethodName)
+    {
+        Assert::true(
+            $this->showPage->hasShipment($shippingMethodName),
+            sprintf('Cannot find shipment "%s".', $shippingMethodName)
+        );
+    }
+
+    /**
+     * @Then I should see :paymentMethodName payment
+     */
+    public function iShouldSeePayment($paymentMethodName)
+    {
+        Assert::true(
+            $this->showPage->hasPayment($paymentMethodName),
+            sprintf('Cannot find payment "%s".', $paymentMethodName)
+        );
+    }
+
+    /**
+     * @Then /^I should see (\d+) items in the list$/
+     */
+    public function iShouldSeeItemsInTheList($amount)
+    {
+        $itemsCount = $this->showPage->countItems();
+
+        Assert::eq(
+            $amount,
+            $itemsCount,
+            sprintf('There should be %d items in the list, but get %d.', $amount, $itemsCount)
+        );
+    }
+
+    /**
+     * @Then I should see the product named :productName in the list
+     * @Then the product named :productName should be in the items list
+     */
+    public function theProductShouldBeInTheItemsList($productName)
+    {
+        Assert::true(
+            $this->showPage->isProductInTheList($productName),
+            sprintf('Product %s is not in the item list.', $productName)
+        );
+    }
+
+    /**
+     * @Then the order's items total should be :itemsTotal
+     */
+    public function theOrdersItemsTotalShouldBe($itemsTotal)
+    {
+        $itemsTotalOnPage = $this->showPage->getItemsTotal();
+
+        Assert::eq(
+            $itemsTotal,
+            $itemsTotalOnPage,
+            sprintf('Items total is "%s", but should be "%s".', $itemsTotalOnPage, $itemsTotal)
+        );
+    }
+
+    /**
+     * @Then the order's total should be :total
+     */
+    public function theOrdersTotalShouldBe($total)
+    {
+        $totalOnPage = $this->showPage->getTotal();
+
+        Assert::eq(
+            $total,
+            $totalOnPage,
+            sprintf('Total is "%s", but should be "%s".', $totalOnPage, $total)
+        );
+    }
+
+    /**
+     * @Then the order's shipping charges should be :shippingCharge
+     */
+    public function theOrdersShippingChargesShouldBe($shippingCharge)
+    {
+        Assert::true(
+            $this->showPage->hasShippingCharge($shippingCharge),
```

Comment:
In this case, we have a list of shipping charges instead of one value as we have in shipping total etc.